lib/package-tile/index.js

```diff
@@ -30,35 +36,50 @@ export default class PackageTile extends Component {
     const totalPriceBeforeDiscount = oldPricePerPerson * totalPassengers;
     return (discountPrice && discountPrice !== '0') ? (totalPriceBeforeDiscount.toLocaleString('da-DK') + ',-') : '';
   }
+  handleOnClick () {
+    this.setState({
+      visible: false
```

Comment:
@jackcarlisle if you instead use `display:none` you will not see the blank spaces on the page identified in https://github.com/numo-labs/isearch-ui/pull/181#issuecomment-222702851

> see: http://stackoverflow.com/questions/133051/what-is-the-difference-between-visibilityhidden-and-displaynone